client/util/format-date.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * OPINIONED date formatter to use browser locale by default
+ */
+
+import { format as dateFnsFormat, Locale, parseISO, formatDistance as dateFnsFormatDistance } from 'date-fns'
+import { enUS, ja } from 'date-fns/locale'
```

Comment:
This reduces bundle size. But on the other hand, it drops many locales support.

I tried to resolve this problem by dynamic import with webpack. Crowi serves JS stuffs with /static path, however webpack’s dynamic loader loads splitted JS stuff with root path.

I think we can put off this problem. If we found correct webpack configuration in future, I will resolve this issue.